midi_app_controller/gui/binds_editor.py

```diff
@@ -110,10 +119,23 @@ def __init__(
 
         self.setLayout(layout)
 
+        # Threads.
         self.thread_list = []
 
-    def _light_up_button(self, button_id: int):
+        # Highlighting.
```

Comment:
Not necessarily in this branch, but I think we should create a superclass for ButtonBinds and KnobBinds - some code is repeated

Reply:
Yes, I agree. There are many code fragments in the project that would benefit from some refactoring.